src/disposable-modal.js

```diff
@@ -29,7 +36,8 @@ export function createCancelableModal(El, props = {}) {
       el = document.createElement("div");
       document.body.appendChild(el);
 
-      ReactDOM.render(
+      const renderer = ReactDOM.createPortal || ReactDOM.render
```

Comment:
This probably doesn't matter, this is an imperative API for rendering a modal and the element is never exposed (even through refs). But if it does matter, should you also update the teardown function? https://github.com/CanopyTax/react-disposable-modal/pull/1/files#diff-ce8028a752a8c66bf4f375d3076e6990R51

Reply:
I was under the impression that it would tear down for either `createPortal` and `render` but I'm not 100% sure. It's one of the things that I would like to test.

Reply:
I'm not sure either, I just noticed in the other PR in primary navbar you had a check for the version of React: https://canopy.githost.io/front-end/red/primary-navbar/merge_requests/175/diffs#c6fbe6a64df719f326e02dbb07c0edd8f31113af_24_24

Reply:
Do you think it matters though? The element is never exposed? This is just a function that you execute to display some content and it is outside of react entirely (react just happens to be used in rendering inside that component). The reason it mattered for the `dialog.component.js` is that people could still get access to the element via refs (which would then be null). It may be unnecessary to do the portal here.

Reply:
Ah, good point. Yeah, it does look like we could probably just use `render` no matter what in this case.